### ISSUE TITLE
在构建时,transform钩子调用babel.transform方法报错

### ISSUE THREAD

**Issue by ysy945** [farm-fe/farm#231]
### Steps to reproduce

1. 在farm.config.ts文件中写插件
2. 在transform钩子中调用babel.transformSync方法报错

原因: 
![image](https://user-images.githubusercontent.com/79794654/232792262-f1e990a4-1c64-41ae-8110-cbf5163e39b0.png)
![image](https://user-images.githubusercontent.com/79794654/232792295-6d521aad-27fb-4a92-92d2-f02e202ea6dc.png)
![image](https://user-images.githubusercontent.com/79794654/232792316-25ad2e04-4b14-4184-8906-828b48abc856.png)
jsReleases require出来没有加default

### What is expected?

能正常运行

### What is actually happening?

![image](https://user-images.githubusercontent.com/79794654/232791675-3bb861fd-7100-45ab-a1dd-ab7bb6611bbc.png)

### System Info

```Shell
mac os
node 18.15.0
```

**Comment by callqh** [comment]
可以提供可以复现的代码吗，我在`js-plugin-vue`中尝试了没有复现该错误
```ts
import { transformSync } from '@babel/core';

...

const result = transformSync(source);

....

```

**Comment by callqh** [comment]
恩 可能已经修复了，回头有时间可以再试一下

**Comment by oblador** [comment]
I ran into similar issues and I believe it’s caused by running browser code in node as reported here https://github.com/farm-fe/farm/issues/234

**Comment by wre232114** [comment]
Fixed in #238
